escrow/tests/functions/create_escrow.rs

```diff
@@ -118,16 +120,16 @@ mod revert {
 
         // TODO: this test likely fails because the param expects an ARRAY of 2 and we provide 0
         //       args. This is likely a panic because of the SDK rather than the test itself
-        create_escrow(
-            defaults.asset_amount,
-            &arbiter_obj,
-            &defaults.asset_id,
-            vec![],
-            buyer.wallet.address(),
-            &seller.contract,
-            defaults.deadline,
-        )
-        .await;
+        // create_escrow(
+        //     defaults.asset_amount,
+        //     &arbiter_obj,
+        //     &defaults.asset_id,
+        //     vec![],
+        //     buyer.wallet.address(),
+        //     &seller.contract,
+        //     defaults.deadline,
+        // )
+        // .await;
```

Comment:
```suggestion
        create_escrow(
            defaults.asset_amount,
            &arbiter_obj,
            &defaults.asset_id,
            vec![],
            buyer.wallet.address(),
            &seller.contract,
            defaults.deadline,
        )
        .await;
```

Reply:
Actually I can't do this because the create escrow function expects an array or 2 Assets and not a vec.  I think it makes the most sense to comment the code out instead of having code that passes 2 assets.

Reply:
Nvm I'm going to change the contract to use a vec and update the tests accordingly